modules/installation-configuration-parameters.adoc

```diff
@@ -807,12 +807,19 @@ Optional AWS configuration parameters are described in the following table:
 |The AWS AMI used to boot compute machines for the cluster. This is required for regions that require a custom {op-system} AMI.
 |Any published or custom {op-system} AMI that belongs to the set AWS region. See _{op-system} AMIs for AWS infrastructure_ for available AMI IDs.
 
+|compute:
+  platform:
+    aws:
+      iamProfile:
+|The name of the IAM instance profile that you use for the machine. If you want the installation program to create the IAM instance profile for you, do not select iamProfile. You can specify iamProfile or iamRole, but you cannot specify both.
```

Comment:
Field name in backticks. Also usually good to qualify what it is. field/parameter, etc. 

```suggestion
|The name of the IAM instance profile that you use for the machine. If you want the installation program to create the IAM instance profile for you, do not use the `iamProfile` field. You can specify either the `iamProfile` or `iamRole` field, but you cannot specify both.
```

I tend to prefer 'field', but looking at this document, 'parameter' is used more often, so that's probably the better choice.